CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.4.0] - 2022-02-15
+
+- Use the latest API version, `v1`
+
```

Comment:
We need to add this "airlines logo API" as well:
https://github.com/duffelhq/duffel-api-ruby/pull/55

Reply:
Done!